event/src/main/java/com/bazaarvoice/emodb/event/db/astyanax/AstyanaxEventReaderDAO.java

```diff
@@ -566,7 +566,7 @@ public void set(int value) {
     protected SlabFilter getSlabFilterSince(final Date since, final String channel) {
         return new SlabFilter() {
             boolean foundStartingSlab = false;
-            final UUID sinceUUID = TimeUUIDs.uuidForTimestamp(since);
+            final UUID sinceUUID = TimeUUIDs.uuidForTimestamp(Date.from(since.toInstant().minus(Constants.SLAB_ROTATE_TTL)));
```

Comment:
Add a 5 minutes more buffer time to SLAB_ROTATE_TTL :?

Reply:
can't hurt 👍